java/java.lsp.server/test/unit/src/org/netbeans/modules/java/lsp/server/protocol/ServerTest.java

```diff
@@ -1604,6 +1604,21 @@ public void showMessage(MessageParams params) {
             public void logMessage(MessageParams arg0) {
                 throw new UnsupportedOperationException("Not supported yet.");
             }
+
+            @Override
+            public CompletableFuture<String> createTextEditorDecoration(DecorationRenderOptions params) {
+                throw new UnsupportedOperationException("Not supported yet.");
```

Comment:
`TestProgressHandlerTest` is using `fail()` while the other tests are using `UnsupportedOperationException` - I guess I like `fail()` more.

Reply:
O.K.